aten/src/ATen/native/native_functions.yaml

```diff
@@ -2959,18 +2959,18 @@
   variants: function, method
   device_guard: False
 
-- func: sum(Tensor self, *, ScalarType? dtype=None) -> Tensor
```

Comment:
I believe I need to remove this schema because it would be ambiguous with the updated `sum.dim_IntList` schema. If it does turn out to be necessary to remove it, I'll clean up the related commented code.

Reply:
This seems like a strange schema to remove. The code edited below, for example, is much more verbose without it. I think we'll want to investigate solutions that preserve it more.